src/template.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.1/jquery.min.js"></script>
+
+    <title>%= htmlWebpackPlugin.options.title %</title>
+  </head>
+  <body>
+    <main class="container">
+      <div>
+        <h1 for="cars">ToDo List:</h1>
+        <img id="recyclImg" src="./assets/refresh.png" alt="refresh todo list" />
+      </div>
```

Comment:
Great work so far, but I think for HTML5 semantic purposes, you could consider using a ,header. tag here instead of div